src/projects/create_new_project.rst

```diff
@@ -1,8 +1,18 @@
 Create a new project
 --------------------
 
-SKA developers are allowed to create new repositories under the SKA-telescope 
-organisation git server.
+The SKA code repositories are all stored on the SKA Gitlab account, on `gitlab.com/ska-telescope <http://gitlab.com/ska-telescope>`_.
+The SKA's repositories on Gitlab have to be created by a member of the Systems team.
+The process is as follows: go to the Slack channel `#team-system-support <https://skasoftware.slack.com/messages/CEMF9HXUZ>`_ and ask for a new repository to be created. Choose the name well (see below). Specify whether the project should be *public* (i.e. anyone can see it), *private* (i.e. accessible to anyone in the SKA Organisation), or *ip protected* (i.e. only for the specific user/team). Ask for any permissions to be granted to specific users.
+
+You will be given Maintainer privileges on this project. For more information about permissions on Gitlab, go to `https://docs.gitlab.com/ee/user/permissions.html <https://docs.gitlab.com/ee/user/permissions.html>`_.
```

Comment:
Add something about the possiblity to add users to the proejct as a result of being maintainer.